upload/admin/view/template/extension/module/smaily_for_opencart.tpl

```diff
@@ -327,6 +325,85 @@
                 </div>
               </div>
             </div>
+            <!-- RSS -->
+            <div id="section4" class="tab-pane fade in">
+              <div class="form-group">
+                <label class="col-sm-2 control-label" for="rss-category"><?php echo $rss_category_title ?></label>
+                <div class="col-sm-10">
+                  <select name="smaily_for_opencart_rss_category" id="rss-category" class="form-control smaily-rss-options">
+                    <option value=''>All Products</option>
```

Comment:
Formatting: mixing single and double quotes in HTML.

Reply:
Replaced with double quotes.